apps/src/gamelab/GameLab.js

```diff
@@ -325,6 +334,7 @@ GameLab.prototype.init = function (config) {
       <GameLabView
         showFinishButton={finishButtonFirstLine && showFinishButton}
         onMount={onMount}
+        onToggleDebugSprites={onToggleDebugSprites}
```

Comment:
There's an antipattern here - a major benefit of putting this property in Redux is that React components can use it and modify it without having to pass a callback down through several layers of view.

Instead what I'd do is in `setupReduxSubscribers()` I'd have `GameLab` watch for `runState.isDebug